ql/src/java/org/apache/hadoop/hive/ql/plan/PlanUtils.java

```diff
@@ -26,14 +26,12 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Comparator;
-import java.util.EnumSet;
```

Comment:
If we do not change the file, we might want to refrain even from cleaning up the imports.

Reply:
Sure, reverted it